server/index.js

```diff
@@ -13,7 +13,6 @@ app.use(bodyparser.urlencoded({ extended: true }));
 
 app.get('/api/rand', (req, res) => {
```

Comment:
You may want a better name for this api endpoint - it's very vague!

Reply:
good point, I'll add switch it over to something more descriptive